frontend/tests/features/home/homePage.test.ts

```diff
@@ -11,62 +11,116 @@ describe("Homepage", () => {
     expect(await page.getByTestId("logo").count()).toBe(2);
     await expect(page.getByTestId(COLLECTION_LINK)).toBeVisible();
 
-    // check page scroll
-    expect(await scrollToPageBottom(page)).toBeTruthy;
+    await isPageScrollableToSeeSiteMap(page);
```

Comment:
extract util function to check if page is scrollable via checking if we can scroll to the bottom to see the sitemap

Reply:
Oh nice! I didn't know they have released this. Thanks